aggregate_logging_sizing.adoc

```diff
@@ -0,0 +1,283 @@
+ https://bugzilla.redhat.com/show_bug.cgi?id=1292447[issue].
+
+A test was performed on a 20 node cluster each with 50 processes logging at 15KB of log data every minute, during 5 minutes. The below messages started to appear on the individual fluentd pods.
+
+----
+$ tail /var/log/messages
+
+Aug 17 11:27:09 ip-172-31-24-231 root: woT3LfOb19xgjtXJEjoblhzJNhwWI8jKPwkPU5HhK7w1bY22ZmvqUWE4Bml71rTkImUjcOzZSWaJ7I3RJTmhQrczF7E1Q44Uy1aVAdkJobPmhsxS7FAWKwNE5GKsp731HccqE8QH9mmaejW3WFc0vMdMZt4ftgrDYQEdcJnzN1i8Mgri673ZcTeOqlTfr3IDy33UWvjUI722djPBpBDRZxyv6ozBex6uMVEQ4tfjkqNB77mXCoRq1syTtSNMlG65
+Aug 17 11:27:09 ip-172-31-24-231 rsyslogd-2177: imjournal: begin to drop messages due to rate-limiting
+----
+
+
+The below indicates journald corruption.
+
+----
+$ dmesg
+
+[251980.030113] systemd-journald[399]: Failed to write entry (13 items, 411 bytes), ignoring: Cannot assign requested address
+[251980.944631] systemd-journald[399]: Failed to write entry (13 items, 411 bytes), ignoring: Cannot assign requested address
+[251981.032716] systemd-journald[399]: Failed to write entry (14 items, 423 bytes), ignoring: Cannot assign requested address
+[251981.953142] systemd-journald[399]: Failed to write entry (13 items, 411 bytes), ignoring: Cannot assign requested address
+
+$ journalctl --verify
+2b6edf8: invalid object
+
+File corruption detected at /run/log/journal/f9370ed252a14f73b014c1301a9b6d1b/system.journal:2b6edf8 (of 50331648 bytes, 90%).
+
+FAIL: /run/log/journal/f9370ed252a14f73b014c1301a9b6d1b/system.journal (Cannot assign requested address)
+PASS: /run/log/journal/f9370ed252a14f73b014c1301a9b6d1b/system@63c069f54f8d48adada6ef72aad6cd2b-0000000000019d1e-00053a3c41130189.journal
+PASS: /run/log/journal/f9370ed252a14f73b014c1301a9b6d1b/system@63c069f54f8d48adada6ef72aad6cd2b-0000000000000001-00053a1b1d414d96.journal
+PASS: /run/log/journal/8cde4877b4ec4bc5962d492606547ac1/system.journal
+----
+
+*Scaling up* +
+
+As you scale up your project, the default logging environment might need some adjustments.
+After upgrading to `systemd-219-22.el7.x86_64`, we've added:
+
+----
+$IMUXSockRateLimitInterval 0
+$IMJournalRatelimitInterval 0
+----
+
+to `/etc/rsyslog.conf` and:
+
+----
+# Disable rate limiting
+RateLimitInterval=1s
+RateLimitBurst=10000
+Storage=volatile
+Compress=no
+MaxRetentionSec=5s
+----
+
+to `/etc/systemd/journald.conf`. 
+
+Now restart the services.
+
+----
+$ systemctl restart systemd-journald.service
+$ systemctl restart rsyslog.service
+----
+
+These settings accommodate for the 'bursty' nature of uploading in bulk. Keep in mind that these tunables are likely to stress the system logging daemons.
+
```

Comment:
this keep in mind sentence has to be clarified.  perhaps

"After removing the rate limit, you may see increased CPU utilization on the system logging daemons as it processes any messages that would have previously been throttled."